run-checks

```diff
@@ -135,6 +126,21 @@ missing_interface_spread_test() {
                 search='plugs: \[ autopilot-introspection \]'
                 ;;
         esac
+
+        # check if a standalone test already exists and that it at least
+        # connects and disconnects the interface
+        dedicated_test=$(find tests/main/ -maxdepth 1 -name "interfaces-$iface")
+        if [ -n "$dedicated_test" ]; then
```

Comment:
Bit of a nitpick but it would be great if we could accumulate all the tests that don't pass this check instead of failing on the first, having to run the script again, fail on the next, etc. But really after this cull, this check would probably only fail when we add new tests and we almost never add more than one at a time so it might not be worth the effort